src/main/java/net/dries007/tfc/CommonEventHandler.java

```diff
@@ -147,6 +194,33 @@ public static void onUseHoe(UseHoeEvent event)
 
     }
 
+
+    @SubscribeEvent
+    public static void onLivingHurt(LivingHurtEvent event)
+    {
+        //Starvation should not be scaled, you receive damage based on % anyway.
+        if (event.getEntityLiving() instanceof EntityPlayer && !(event.getSource() == DamageSource.STARVE))
+        {
+            EntityPlayer player = (EntityPlayer) event.getEntityLiving();
+            IPlayerData cap = event.getEntityLiving().getCapability(CapabilityPlayer.CAPABILITY_PLAYER_DATA, null);
+            if (cap != null)
+            {
+                float damage = DamageManager.rescaleDamage(event.getAmount(), 20, ConfigTFC.GENERAL.playerBaseHealth);
```

Comment:
As I said in discord - there is **no reason** to scale damage up to 1000 here. In the same vein, having a config value for this is confusing and counterproductive - It should only be used for display purposes.

This still needs to rescale damage, yes, but instead of doing `finalDamage = damage * (1000/20) * modifiers * (20/actualMaxHp)`, it should just become `finalDamage = damage * modifiers * maxHpModifier`

Reply:
since i detected: instead of `finalDamage = damage * modifiers * maxHpModifier` => `finalDamage = damage * modifiers / maxHpModifier`
As HP is the inverse of damage.